exonum-light-client/src/test/java/com/exonum/client/ExonumHttpClientIntegrationTest.java

```diff
@@ -83,16 +84,7 @@ void submitTransactionTest() throws InterruptedException {
     // Assert request params
     RecordedRequest recordedRequest = server.takeRequest();
     assertThat(recordedRequest.getMethod(), is("POST"));
-    assertThat(recordedRequest.getPath(), is(SUBMIT_TRANSACTION));
-
-    // Assert request encoding
-    String json = recordedRequest.getBody().readUtf8();
-    SubmitTxRequest actualRequest = json().fromJson(json, SubmitTxRequest.class);
-    String encodedTxMessage = actualRequest.getBody();
-    byte[] actualMessageBytes = HEX_ENCODER.decode(encodedTxMessage);
-    TransactionMessage actualTxMessage = TransactionMessage.fromBytes(actualMessageBytes);
-
-    assertThat(actualTxMessage, is(txMessage));
```

Comment:
Don't we longer check that we sent a properly encoded transaction?

Reply:
we rely on `Gson` instance here. But I agree, let's have some checks in case of regression.